app/src/main/java/com/pcsa/views/reporting/ContactPostStaff.java

```diff
@@ -41,14 +45,39 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_reporting_contact_post_staff);
 
-        contact_pcmo = (Button) findViewById(R.id.post_staff_pcmo);
-        contact_ssm = (Button) findViewById(R.id.post_staff_ssm);
-        contact_sarl = (Button) findViewById(R.id.post_staff_sarl);
-        current_location = (TextView) findViewById(R.id.post_staff_current_location);
+        locationDetails.put(getResources().getString(R.string.loc1_name), new LocationDetails(getResources().getString(R.string.loc1_name), getResources().getString(R.string.loc1_pcmo), getResources().getString(R.string.loc1_ssm), getResources().getString(R.string.loc1_sarl)));
+        locationDetails.put(getResources().getString(R.string.loc2_name), new LocationDetails(getResources().getString(R.string.loc2_name), getResources().getString(R.string.loc2_pcmo), getResources().getString(R.string.loc2_ssm), getResources().getString(R.string.loc2_sarl)));
+        locationDetails.put(getResources().getString(R.string.loc3_name), new LocationDetails(getResources().getString(R.string.loc3_name), getResources().getString(R.string.loc3_pcmo), getResources().getString(R.string.loc3_ssm), getResources().getString(R.string.loc3_sarl)));
+
```

Comment:
We are not getting data from a server at the moment as discussed. Therefore, data is added to XML and being retrieved from there :)